### DISCUSSION TITLE
Welcome to las2peer Discussions!

### DISCUSSION THREAD

**Discussion by pedela** [rwth-acis/LAS2peer#80] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by HackIntoYourHeart** [comment]
I finished building the las2peer project but I don't know how to run it? 
I want to run core and front-end but I don't see any guides to run it ?
